crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -372,7 +375,10 @@ impl Schedule {
 
     /// Configures a collection of system sets in this schedule, adding them if they does not exist.
     #[track_caller]
-    pub fn configure_sets(&mut self, sets: impl IntoSystemSetConfigs) -> &mut Self {
+    pub fn configure_sets<Marker>(
+        &mut self,
+        sets: impl IntoNodeConfigs<InternedSystemSet, Marker>,
+    ) -> &mut Self {
```

Comment:
```suggestion
    pub fn configure_sets<M>(
        &mut self,
        sets: impl IntoNodeConfigs<InternedSystemSet, M>,
    ) -> &mut Self {
```
For functions that use the marker generic, we use `M` instead of the full `Marker` name.